<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultsSummaryPage</class>
 <widget class="QWidget" name="ResultsSummaryPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>502</height>
   </rect>
  </property>
  <property name="toolTip">
   <string>Summary of the profile data analysis.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="summaryScrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="summaryScrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>287</width>
        <height>679</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="parserErrorsBox">
         <property name="toolTip">
          <string>Errors occurred parsing the data input. You may need to change the search paths to resolve errors for missing ELF files.</string>
         </property>
         <property name="title">
          <string>Parser Errors</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QTreeView" name="parserErrorsView">
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <property name="uniformRowHeights">
             <bool>true</bool>
            </property>
            <property name="headerHidden">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="summaryGroupBox">
         <property name="title">
          <string>Summary</string>
         </property>
         <layout class="QVBoxLayout" name="summaryBoxLayout">
          <item>
           <widget class="QLabel" name="summaryLabel">
            <property name="toolTip">
             <string notr="true"/>
            </property>
            <property name="statusTip">
             <string notr="true"/>
            </property>
            <property name="whatsThis">
             <string notr="true"/>
            </property>
            <property name="accessibleName">
             <string notr="true"/>
            </property>
            <property name="accessibleDescription">
             <string notr="true"/>
            </property>
            <property name="text">
             <string notr="true">summary text</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="toolTip">
          <string>The hotspots found in the profile, i.e. the code that contributed the most samples.</string>
         </property>
         <property name="title">
          <string>Top Hotspots</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="eventSourceLabel">
               <property name="text">
                <string>Event Source:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="eventSourceComboBox"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QTreeView" name="topHotspotsTableView">
            <property name="minimumSize">
             <size>
              <width>1</width>
              <height>150</height>
             </size>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <property name="uniformRowHeights">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Top Hotspots per File</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Event Source:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="eventSourceComboBox_2"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QTreeView" name="topLibraryTreeView">
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <property name="uniformRowHeights">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="systemInfoGroupBox">
         <property name="title">
          <string>System Information</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="systemInfoLabel">
            <property name="toolTip">
             <string notr="true"/>
            </property>
            <property name="statusTip">
             <string notr="true"/>
            </property>
            <property name="whatsThis">
             <string notr="true"/>
            </property>
            <property name="accessibleName">
             <string notr="true"/>
            </property>
            <property name="accessibleDescription">
             <string notr="true"/>
            </property>
            <property name="text">
             <string notr="true">system info</string>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
